### ISSUE TITLE
Create fixture for ballot app data

### ISSUE THREAD

**Issue by adborden** [caciviclab/disclosure-backend#46]
Should include the an auth group "Ballot Editors" which can edit all the ballot content. You should be able to dump this from the production db (give it a few weeks).

**Comment by bcipolli** [comment]
Is this different from #41 ?

**Comment by adborden** [comment]
Yeah, this is different than the backup story. The backup story is about preserving the production DB in case of a disaster.

This story is more about fleshing out [ballot data](https://github.com/caciviclab/disclosure-backend/blob/master/ballot/fixtures/ballot_data.json) for your dev instance so that when you're running the app locally, the app is usable. I suppose there is a distinction here. Since the ballot source data is currently not fetchable (as opposed to netfile, for example), it should be loaded as a fixture.

**Comment by adborden** [comment]
If we had data on http://admin.caciviclab.org/, we could add a management command that pulls from there and skip the json fixture data.
